Cannot find module cypress-image-diff-js/dist/plugin

I'm trying to run the plugin for the first time and I encounter the following problem, could you help me, please?

Error:
```
Error: Cannot find module '/Users/caiquecoelho/Documents/Projects/openid_server_code/node_modules/cypress-image-diff-js/dist/plugin'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1098:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1091:15)
    at resolveExports (node:internal/modules/cjs/loader:567:14)
    at Function.Module._findPath (node:internal/modules/cjs/loader:636:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1063:27)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/Users/caiquecoelho/Library/Caches/Cypress/13.6.0/Cypress.app/Contents/Resources/app/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:119:18)
    at Object.<anonymous> (/Users/caiquecoelho/Documents/Projects/openid_server_code/cypress.config.ts:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module.m._compile (/Users/caiquecoelho/Library/Caches/Cypress/13.6.0/Cypress.app/Contents/Resources/app/node_modules/ts-node/dist/index.js:857:29)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/caiquecoelho/Library/Caches/Cypress/13.6.0/Cypress.app/Contents/Resources/app/node_modules/ts-node/dist/index.js:859:16)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Function.Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:119:18)
    at loadFile (/Users/caiquecoelho/Library/Caches/Cypress/13.6.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/child/run_require_async_child.js:89:14)
    at EventEmitter.<anonymous> (/Users/caiquecoelho/Library/Caches/Cypress/13.6.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/child/run_require_async_child.js:116:38)
    at EventEmitter.emit (node:events:517:28)
    at EventEmitter.emit (node:domain:489:12)
```

cypress.config.ts
```
import { defineConfig } from 'cypress';
import getCompareSnapshotsPlugin from 'cypress-image-diff-js/plugin';

export default defineConfig({
  e2e: {
    setupNodeEvents(on, originalConfig) {
      const config = getCompareSnapshotsPlugin(on, originalConfig);
      require('@cypress/grep/src/plugin')(config);
      getCompareSnapshotsPlugin(on, config);
      return require('./cypress/plugins/index.js')(on, config);
    },
    experimentalRunAllSpecs: true,
    specPattern: 'cypress/e2e/**/*.{js,jsx,ts,tsx}',
    baseUrl: 'http://localhost:9000',
    retries: {
      runMode: 1,
      openMode: 0,
    },
    experimentalMemoryManagement: true,
    numTestsKeptInMemory: process.env.CI ? 0 : 10,
  },
  chromeWebSecurity: false,
  defaultCommandTimeout: 10000,
  requestTimeout: 25000,
  responseTimeout: 25000,
  env: {
    grepFilterSpecs: true,
    grepOmitFiltered: true
  }
});

```


**System:**
Mac Os 14.1 (23B74)

**Cypress: 13.6.0**

--- a/package.json
+++ b/package.json
@@ -5,11 +5,11 @@
   "exports": {
     "./command": {
       "types": "./types/command.d.ts",
-      "default": "./dist/command"
+      "default": "./dist/command.js"
     },
     "./plugin": {
       "types": "./types/plugin.d.ts",
-      "default": "./dist/plugin"
+      "default": "./dist/plugin.js"
     }
   },
   "files": ["dist", "types"],

